File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.cc
#define _CRT_SECURE_NO_WARNINGS  
#include <stdio.h>
int main()
{
	int num = 0;
	scanf("%d", &num);
	int n = num;
	int add = 0;
	int cnt = 0;

	do {
		add = num / 10 + num % 10;
		num = add % 10 + num % 10 * 10;
		cnt++;
	} while (n != num);

	printf("%d",cnt);
}

File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.java

import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in); 
		
		int num = sc.nextInt();
		int n = num;
		int add = 0; 
		int cnt = 0; 
		
		do {
			add = num/10 + num%10; 
			num = add%10 + num%10*10; 
			cnt++; 
		}while(n != num);
		
		System.out.print(cnt);
	}

}